README.md

```diff
@@ -45,13 +45,16 @@ or the below command on Linux or Mac OS:
 python3 main.py
 ```
 
+## Out of repo contributions
+- **Irish large preprocessed data**
+[This repo](https://github.com/Annoyed-Raven/Irish_processing) contains pure Irish words extracted from several sources. Some of the main text files before preprocessing contained around 1400 non-Irish characters.
+
```

Comment:
@Annoyed-Raven I already added the link here. I suggest you modify the description here if you like to and delete the ```Irish preprocessing raw files``` section that you added.